frontend/src/pages/ViewArticle.js

```diff
@@ -44,6 +44,20 @@ const ViewArticle = () => {
     });
   };
 
+  const searchPractice = (article) => {
+    if (!article) {
+      getArticle().then(({ data }) => {
+        setArticles(
+          data.filter((data) => data.sepractice === selectedPractice)
+        );
+      });
+    } else {
+      getArticle().then(({ data }) => {
+        setArticles(data.filter((data) => data.title === article));
+      });
+    }
+  };
+
```

Comment:
Add error handling for GET request.

```suggestion
  const searchPractice = (article) => {
    if (!article) {
      getArticle()
      .then(({ data }) => {
        setArticles(
          data.filter((data) => data.sepractice === selectedPractice)
        );
      })
      .catch((error) => {
        console.error(error)
      });
    } else {
      getArticle()
      .then(({ data }) => {
        setArticles(data.filter((data) => data.title === article));
      })
      .catch((error) => {
        console.error(error)
      });
    }
  };
```